.github/actions/warm-up-repo/action.yml

```diff
@@ -26,6 +26,7 @@ runs:
       uses: Swatinem/rust-cache@v2
       with:
         workspaces: libs/@blockprotocol/type-system/crate
+        shared-key: "shared"
```

Comment:
Do you know the implication of this if we are making changes to the Rust setup (dependencies or something else) in multiple PRs?

What would happen if we added a new crate?

Reply:
This only sets up for the workspace above, so this would not affect any other crates at all. We need to revisit caching when adding a new crate in any case.

Reply:
And
> Do you know the implication of this if we are making changes to the Rust setup (dependencies or something else) in multiple PRs

Reply:
The cache cannot be shared across multiple PRs, it can only re-use the cache from the base branch.

Reply:
Ah I understand, this is shared between _jobs_ on the same PR (sorry I know you wrote that in the PR description I just didn't understand).